### ISSUE TITLE
ARM support

### ISSUE THREAD

**Issue by fabiospampinato** [notable/notable#1062]

**Comment by hardwareadictos** [comment]
So, it's on your thoughts to bring an arm version? 😁

**Comment by fabiospampinato** [comment]
Yes.

**Comment by hardwareadictos** [comment]
> Yes.

Thank you! I really think it's a nice idea. I have a Chromebook and various PIs, met me know if you need a tester 😊

**Comment by fabiospampinato** [comment]
I'll let you know when I'm ready to start testing this 👍 in the mean time: can you not run any of the currently available linux packages without setting up a virtual machine?

**Comment by hardwareadictos** [comment]
> I'll let you know when I'm ready to start testing this  in the mean time: can you not run any of the currently available linux packages without setting up a virtual machine?

Sorry I don't understand what are you referring to. I'm using both Linux (.Deb) and Windows (.exe) versions natively on different devices. This app has became my principal markdown note taking system.

**Comment by fabiospampinato** [comment]
I was referring to the Chromebook, can you run the app today without major hassles on that?

**Comment by hardwareadictos** [comment]
> I was referring to the Chromebook, can you run the app today without major hassles on that?

I can't, I have an Asus C101. Which has an arm64 processor, so I cannot run any of the official supported pacakges, cannot also run AppImage as it throws exec format errors.

**Comment by fabiospampinato** [comment]
I see, I think there are a lot of chromebooks with ARM around, I'll try to prioritize this more.

**Comment by hardwareadictos** [comment]
> I see, I think there are a lot of chromebooks with ARM around, I'll try to prioritize this more.

Thanks for taking this archs seriously. Feel free for asking requesting me whatever you need 😃

**Comment by bvitaliyg** [comment]
@fabiospampinato any updates of this? It is important since Apple continue its migration for Apple Silicon

**Comment by lillycat332** [comment]
yeah, this would be great considering Electron and similar web browser frameworks run pretty bad under Rosetta compared to other apps

**Comment by sandronoel** [comment]
So.... this is pretty much dead.
